### ISSUE TITLE
Potential dependency conflicts between awslimitchecker and botocore

### ISSUE THREAD

**Issue by NeolithEra** [jantman/awslimitchecker#454]
Hi, as shown in the following full dependency graph of  **_awslimitchecker_**, **_awslimitchecker_** requires **_botocore  >=1.12.175_** ， while the installed version of **_boto3_**(1.10.45) requires _**botocore >=1.13.45,<1.14.0**_.

According to Pip's _“first found wins”_ installation strategy, _**botocore 1.13.45**_ is the actually installed version.  

Although the first found package version _**botocore 1.13.45**_ just satisfies the later dependency constraint （_**botocore >=1.13.45,<1.14.0**_), it will lead to a build failure once developers release a newer version of botocore.

### Dependency tree--------
```
awslimitchecker - 8.0.0
| +- boto3(install version:1.10.45 version range:>=1.9.175)
| | +- botocore(install version:1.13.45 version range:>=1.13.45,<1.14.0)
| | | +- docutils(install version:0.15.2 version range:>=0.10,<0.16)
| | | +- jmespath(install version:0.9.4 version range:<1.0.0,>=0.7.1)
| | +- jmespath(install version:0.9.4 version range:<1.0.0,>=0.7.1)
| | +- s3transfer(install version:0.2.1 version range:>=0.2.0,<0.3.0)
| | | +- botocore(install version:1.13.45 version range:<2.0.0,>=1.12.36)
| | | | +- docutils(install version:0.15.2 version range:>=0.10,<0.16)
| | | | +- jmespath(install version:0.9.4 version range:<1.0.0,>=0.7.1)
| +- botocore(install version:1.13.45 version range:>=1.12.175)
| | +- docutils(install version:0.15.2 version range:>=0.10,<0.16)
| | +- jmespath(install version:0.9.4 version range:<1.0.0,>=0.7.1)
| +- python-dateutil(install version:2.8.1 version range:>=2.4.2)
| +- pytz(install version:2019.3 version range:*)
| +- termcolor(install version:1.1.0 version range:>=1.1.0)
| +- urllib3(install version:1.25.6 version range:*)
| +- versionfinder(install version:1.0.0 version range:>=0.1.1) 
```

Thanks for your attention.
Best,
Neolith

**Comment by NeolithEra** [comment]
### Suggested Solution

1.	Fix your direct dependencies to be **_botocore <1.14.0_**.
2.	Ask your upstream project **_boto3_** to lose the version range of **_botocore_** to be **_>=1.13.45_**, and **_s3transfer_** to lose the version range of **_botocore_** to be **_>=1.12.36_**.

@jantman Which solution do you prefer, 1 or 2?
Please let me know your choice. May I pull a request to solve this issue?

**Comment by jantman** [comment]
This has been released in [8.0.1](https://github.com/jantman/awslimitchecker/releases/tag/8.0.1) and is now [live on PyPI](https://pypi.org/project/awslimitchecker/). Thank you!
